client/src/tests/stores/Setup.test.ts

```diff
@@ -0,0 +1,31 @@
+import SetupStore from "../../stores/SetupStore";
+import SetupActions from "../../actions/SetupActions";
+
+describe('Data Source: Samples', () => {
+
+  this.timeout = 10000;
+  beforeAll(() => {
+  })
+
+  it('Testing SetupActions', (done) => {
+
+    let checkState = (state) => {
+      expect(state).toHaveProperty('admins');
```

Comment:
Is it possible to compare it to an object instead (or better yet a snapshot - wouldn't require almost any work, and would automatically save your object "image" to compare it later)